# Testing merge conflicts

Start by importing the required packages:


```{python}
import pandas as pd
```

Let's create a sample dataset to work with:

```{python}
# Create Star Wars DataFrame
star_wars_df = pd.DataFrame({
    "name": ["Luke Skywalker", "Darth Vader", "Leia Organa", "Han Solo", "Yoda", "Chewbacca"],
    "height": [172, 202, 150, 180, 66, 228],
    "mass": [77, 136, 49, 80, 17, 112],
    "species": ["Human", "Human", "Human", "Human", "Yoda's species", "Wookiee"]
})
star_wars_df
```

And do something with it:

```{python}
# Your code goes here
```

<<<<<<< HEAD
```{python}
star_wars_df['species'].unique()
star_wars_df.head()
```
=======
xD :P /.\ .______.
>>>>>>> e448b2b324174a752bc2729c9fd04b82a800e0e5


conflict